addons/website_event_meet/static/src/js/snippets/options.js

```diff
@@ -28,10 +50,10 @@ options.registry.WebsiteEvent.include({
     /**
      * @override
      */
-    async _computeWidgetState(methodName, params) {
+    _computeWidgetState(methodName, params) {
         switch (methodName) {
             case 'allowRoomCreation': {
-                return this._getRpcData('meeting_room_allow_creation');
+                return this.allowRoomCreation;
```

Comment:
When you're doing this, you're overriding the method "allowRoomCreation" which means you cannot enable or disable the option anymore, just rename it and it should be fine 😄

Reply:
Indeed, my bad, I renamed it :sweat_smile: